### PR TITLE
Postman Tests for Rule008

### PR DESCRIPTION
**Pull request by vorsterk**

### CODE REVIEW

**Review by johanfol — APPROVED** [review]

File: Rule-008 Tests.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,2280 @@
...
+							"}]\r",
+							"\r",
+							"const transactionGraph = [\r",
+							"{\r",
+							"    _key: \"d923682a97089a841615181e58355b8eeee108007f7898\",\r",
+							"    _from: \"accounts/24058ebbdd500fe2307cdd59186e0abcd9726de434ae492dbe9d1e12822914aa\",\r",
+							"    _to: \"accounts/352afab0eed3e035ff73cec42277d4a9f66f289cee9c16dc6bca02eba4d91566\",\r",
+							"    Amt: 2000,\r",
+							"    TxTp: \"pacs.008.001.10\",\r",
+							"    TxSts: \"ACCC\",\r",
```
Comment (uche-lextego) [inline Rule-008 Tests.postman_collection.json]: Welldone @vorsterk on this postman tests. Why does your Pacs008 edges have Txsts????

File: Rule-008 Tests.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,2280 @@
...
+							"    }\r",
+							"}]\r",
+							"\r",
+							"const transactionGraph = [{\r",
+							"    _key: \"d923682a97089a8416151ad1e58355b8eeee108007\",\r",
+							"    _from: \"accounts/352afab0eed3e035ff73cec422c7d4a9f66f289cee9c16dc6bca02eba4d91566\",\r",
+							"    _to: \"accounts/24058ebbdd500fe2307cdd55186e0abcd9726de434ae492dbe9d1e12822914aa\",\r",
+							"    Amt: 9000,\r",
+							"    TxTp: \"pacs.008.001.10\",\r",
+							"    TxSts: \"ACCC\",\r",
```
Comment (uche-lextego) [inline Rule-008 Tests.postman_collection.json]: Here too????

**Review by uche-lextego — CHANGES_REQUESTED** [review]
Hello @vorsterk , welldone on this task. Please find the comments I made on removing Txsts from your pacs008 edges seeds.

**Review by uche-lextego — APPROVED** [review]
LGTM

**Pull request merged**